sonaranalyzer-dotnet/src/SonarAnalyzer.CFG/ControlFlowGraph/CSharpControlFlowGraphBuilder.cs

```diff
@@ -52,7 +51,7 @@ internal CSharpControlFlowGraphBuilder(CSharpSyntaxNode node, SemanticModel sema
 
         protected override void PostProcessGraph()
         {
-            FixJumps(this.GotoJumpBlocks, this.LabeledStatements.ToDictionary(e => e.Key, e => (Block)e.Value));
+            FixJumps(GotoJumpBlocks, LabeledStatements.ToDictionary(e => e.Key, e => (Block)e.Value));
```

Comment:
There are a few remaining `this.` updates that we should preserve for consistency in this file